frontend/src/teams/team/team-route.tsx

```diff
@@ -32,17 +33,24 @@ type Params = {
 };
 
 const TeamRoute = () => {
-  const api = teamsAPI();
+  const teamsApi = teamsAPI();
+  const playersApi = playersAPI();
   const params = useParams<Params>();
   const [team, setTeam] = useState<Team>();
 
+  const deletePlayer = (teamId: number, playerId: number) => {
```

Comment:
You are now in TeamRoute, and there you have `team` state, so why you created function `deletePlayer` which needs `teamId`? You can just pass it here.